semtypes/src/main/java/io/ballerina/types/Core.java

```diff
@@ -70,6 +70,9 @@
  */
 public final class Core {
 
+    private static final boolean SEM_PORT_TEST =
+            Boolean.parseBoolean(System.getProperty("ballerina.semtype.port.test"));
```

Comment:
Shall we define this field as a constant

Reply:
This is a temporary field for a hack.(Until we integrate all the types) No point extracting the property name to a separate field as I don't expect it to be re-used.